[{"id":"CVE-2020-26943","purls":["pkg:pypi/blazar@0","pkg:pypi/blazar-dashboard@3.0.0","pkg:pypi/blazar-dashboard@2.0.0","pkg:pypi/blazar-dashboard@0","pkg:pypi/blazar-dashboard@1.0.0","pkg:pypi/blazar-dashboard@1.0.1","pkg:pypi/blazar-dashboard@1.1.0","pkg:pypi/blazar-dashboard@1.2.0","pkg:pypi/blazar-dashboard@1.3.0"],"base_cpe":"cpe:2.3:a:openstack:blazar-dashboard","first_patched_purls":["pkg:pypi/blazar-dashboard@2.0.1","pkg:pypi/blazar-dashboard@3.0.1","pkg:pypi/blazar-dashboard@1.3.1"],"scoreCVSS2":9,"scoreCVSS3":9.9,"vectorCVSS2":"AV:N/AC:L/Au:S/C:C/I:C/A:C","vectorCVSS3":"CVSS:3.1/AV:N/AC:L/PR:L/UI:N/S:C/C:H/I:H/A:H","cwe_ids":[],"severity":"HIGH","published_date":"2020-10-16","last_modified_date":"2020-10-27","description":"An issue was discovered in OpenStack blazar-dashboard before 1.3.1, 2.0.0, and 3.0.0. A user allowed to access the Blazar dashboard in Horizon may trigger code execution on the Horizon host as the user the Horizon service runs under (because the Python eval function is used). This may result in Horizon host unauthorized access and further compromise of the Horizon service. All setups using the Horizon dashboard with the blazar-dashboard plugin are affected. See: CVE-2020-26943.","references":["https://review.opendev.org/755813","https://review.opendev.org/755814","https://review.opendev.org/756064","http://www.openwall.com/lists/oss-security/2020/10/16/5","https://nvd.nist.gov/vuln/detail/CVE-2020-26943","https://github.com/advisories/GHSA-939m-4xpw-v34v","https://launchpad.net/bugs/1895688","https://security.openstack.org/ossa/OSSA-2020-007.html","https://review.opendev.org/755810","https://review.opendev.org/755812"],"patch_links":[],"exploits":[],"patches":[]}]
